### ISSUE TITLE
After creating a map using Point-LIO, can I load the pcd file and just localize it?

### ISSUE THREAD

**Issue by citizen135** [hku-mars/Point-LIO#93]
There doesn't seem to be any related codes.

**Comment by Joanna-HE** [comment]
yes, of course. The related code is in the function of publish_frame_world(). You can turn the save_scan as true.

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Jonghyun Park ***@***.***>
发送时间: Sunday, July 28, 2024 7:58:55 PM
收件人: hku-mars/Point-LIO ***@***.***>
抄送: Subscribed ***@***.***>
主题: [hku-mars/Point-LIO] After creating a map using Point-LIO, can I load the pcd file and just localize it? (Issue #93)


This is an external email.

There doesn't seem to be any related codes.

―

**Comment by migsdigs** [comment]
Hi @Joanna-HE, is it also possible to load the saved map and localize in it?
